### PR TITLE
Add EDS for static endpoints

### PR DESCRIPTION
**Pull request by arkodg**
Fixes: https://github.com/envoyproxy/gateway/issues/787

### CODE REVIEW

File: internal/xds/translator/translator.go [hunk 1]
```diff
@@ -229,6 +218,18 @@ func findXdsCluster(tCtx *types.ResourceVersionTable, name string) *clusterv3.Cl
 	return nil
 }
 
+func addXdsCluster(tCtx *types.ResourceVersionTable, name string, destinations []*ir.RouteDestination, tSocket *corev3.TransportSocket, isHTTP2 bool, isStatic bool) {
```
Comment (zirain) [inline internal/xds/translator/translator.go:221]: IMO, the last two parameters(`isHTTP2` `isStatic`) has a smell of bad code, so you need add comments everytime you call `addXdsCluster`.

Comment (arkodg) [reply internal/xds/translator/translator.go:221]: agree, we should add a `addClusterArgs{}` in a future PR

Comment (zirain) [reply internal/xds/translator/translator.go:221]: can you open a ticket to follow up?

Comment (arkodg) [reply internal/xds/translator/translator.go:221]: raised https://github.com/envoyproxy/gateway/issues/1141

**Pull request merged**